src/gui/scaleChargesDialogFuncs.cpp

```diff
@@ -1,21 +1,19 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 // Copyright (c) 2024 Team Dissolve and contributors
 
+#include "gui/models/scaleChargesDialogModel.h"
 #include "gui/scaleChargesDialog.h"
+#include <QQmlContext>
+#include <QQuickItem>
+#include <QQuickWidget>
 
-ScaleChargesDialog::ScaleChargesDialog(QWidget *parent) { ui_.setupUi(this); }
-ScaleChargesDialog::~ScaleChargesDialog() {}
+ScaleChargesDialog::ScaleChargesDialog(QWidget *parent) : QDialog(parent)
+{
+    QQuickWidget *view = new QQuickWidget(QUrl("qrc:/dialogs/qml/ScaleChargesDialog.qml"), this);
 
-double ScaleChargesDialog::scaleValue() const { return ui_.ScaleSpinBox->value(); }
+    auto root = view->rootObject();
+    model = root->findChild<ScaleChargesDialogModel *>("dialogModel");
 
-void ScaleChargesDialog::on_CancelButton_clicked(bool checked) { reject(); }
-void ScaleChargesDialog::on_ScaleButton_clicked(bool checked)
-{
-    scale_ = true;
-    accept();
-}
-void ScaleChargesDialog::on_ScaleToButton_clicked(bool checked)
-{
-    scaleTo_ = true;
-    accept();
-}
+    QObject::connect(model, SIGNAL(cancelSelection()), this, SLOT(reject()));
+    QObject::connect(model, SIGNAL(acceptSelection()), this, SLOT(accept()));
```

Comment:
```suggestion
    QObject::connect(model, SIGNAL(acceptSelection()), this, SLOT(accept()));
    
    QHBoxLayout *topLeftLayout = new QHBoxLayout;
    topLeftLayout->addWidget(view);
    setLayout(topLeftLayout);
```

Tell the old-style Qt dialog to update the size of the view object when the window is resized.